flarestack/core/results.py

```diff
@@ -707,7 +746,14 @@ def find_disc_potential(self):
 
         sols = []
 
-        for i, y_val in enumerate([y, y_25]):
+        if np.isnan(disc_threshold):
```

Comment:
Maybe, to avoid duplication/prevent future issues with the name `disc_potential_25` being written twice, it could instead be:

```
y_list = [y_25]
out_list = ["disc_potential_25"]

if not np.isnan(disc_threshold):
    y_list.append(y)
    out_list.append("disc_potential")
```

Reply:
I agree, done!